results/i9-12900H/v.semykin/g++-14/run_benchmarks.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
+
+declare -A benchmarks=(
+    ["coro-fibs"]="coro-fibs/coro-fibs"
+    ["exc_ret"]="excret/exc_ret"
+    ["exc_ret_noexc"]="excret/exc_ret_noexc"
+    ["extswap"]="inline/extswap"
+    ["exc_partition"]="noexcept-qsort/exc_partition"
+    ["exc_qsort"]="noexcept-qsort/exc_qsort"
+    ["filter"]="ranges-filter/filter"
+    ["projector"]="ranges-projector/projector"
+    ["virtinh"]="virtual-inherit/virtinh"
+    ["virtinl"]="virtual-inline/virtinl"
+    ["virtual"]="virtual-overhead/virtual"
+    ["virtual-shuffle"]="virtual-overhead/virtual-shuffle"
+)
+
+base_dir="../../../../build/Release"
+
+for benchmark in "${!benchmarks[@]}"; do
```

Comment:
What if I need to run a set of benchmarks, not all of them? Would be nice to add posibility to do so, better yet, benchmarks need to be labeled. Again, how about we reconsider writing this in Python, for instance?

Reply:
bash scripts tend to go unmanagable when you add more and more functionality. They are good when  you need to manipulate output of some linux commands, but they barely support some logic beyond that. I would advise rewriting this in python, you could easily label benchmarks and add CLI options for launching them like in any good test/benchmark framework. The issue with paths, CLI and labeling can be much easier solved with python.

Reply:
Okay, thank you for this advice, I'll rewrite it later